lib/sycamore/sycamore/connectors/file/file_scan.py

```diff
@@ -76,13 +76,15 @@ def __init__(
         paths: Union[str, list[str]],
         *,
         filesystem: Optional[FileSystem] = None,
-        parallelism: Optional[int] = None,
+        parallelism: Optional[str] = None,
+        override_num_blocks: Optional[int] = None,
```

Comment:
I almost wonder if we want to expose this at all. I get that it makes sense as an alternative to parallelism, but it is kind of ray specific and probably hard for people to use. Should we just leave it in `resources_args`?

Reply:
+1

Reply:
We probably don't want to expose it, but at this point we have. I added a comment to consider removing it and updated the documentation to be more clear people probably don't want this.